models/marts/advertising/fct_ad_performance.sql

```diff
@@ -24,7 +24,7 @@ id_creation as (
                     'adset_name',
                     'fb_campaign_name'
                 )}}
-            when platform = 'adwords' then
+            when platform = 'google' then
```

Comment:
my last question is to just confirm that you have ids matching for both adwords and fb between `ad_performance` and `fct_snowplow_sessions`. if you do, this is good to merge!